code/game/gamemodes/scoreboard.dm

```diff
@@ -33,76 +59,70 @@
 					score["escapees"] += 1
 //					player.unlock_medal("100M Dash", 1)
 //				player.unlock_medal("Survivor", 1)
-//				for (var/obj/item/weapon/gnomechompski/G in player.get_contents())
+//				for(var/obj/item/weapon/gnomechompski/G in player.get_contents())
 //					player.unlock_medal("Guardin' gnome", 1)
 
+					var/cashscore = 0
+					var/dmgscore = 0
+
+
+					for(var/obj/item/weapon/card/id/C1 in get_contents_in_object(E, /obj/item/weapon/card/id))
+						cashscore += C1.GetBalance()
 
-	var/cashscore = 0
-	var/dmgscore = 0
-	for(var/mob/living/carbon/human/E in mob_list)
-		cashscore = 0
-		dmgscore = 0
-		var/turf/location = get_turf(E.loc)
-		var/area/escape_zone = locate(/area/shuttle/escape/centcom)
-		if(E.stat != 2 && location in escape_zone) // Escapee Scores
-			for (var/obj/item/weapon/card/id/C1 in get_contents_in_object(E, /obj/item/weapon/card/id))
-				cashscore += C1.GetBalance()
-
-			for (var/obj/item/weapon/spacecash/C2 in get_contents_in_object(E, /obj/item/weapon/spacecash))
-				cashscore += C2.worth
-
-//			for(var/datum/data/record/Ba in data_core.bank)
-//				if(Ba.fields["name"] == E.real_name) cashscore += Ba.fields["current_money"]
-			if (cashscore > score["richestcash"])
-				score["richestcash"] = cashscore
-				score["richestname"] = E.real_name
-				score["richestjob"] = E.job
-				score["richestkey"] = E.key
-			dmgscore = E.bruteloss + E.fireloss + E.toxloss + E.oxyloss
-			if (dmgscore > score["dmgestdamage"])
-				score["dmgestdamage"] = dmgscore
-				score["dmgestname"] = E.real_name
-				score["dmgestjob"] = E.job
-				score["dmgestkey"] = E.key
+					for(var/obj/item/weapon/spacecash/C2 in get_contents_in_object(E, /obj/item/weapon/spacecash))
+						cashscore += C2.worth
+
+//					for(var/datum/data/record/Ba in data_core.bank)
+//						if(Ba.fields["name"] == E.real_name)
+//							cashscore += Ba.fields["current_money"]
+					if(cashscore > score["richestcash"])
+						score["richestcash"] = cashscore
+						score["richestname"] = player.real_name
+						score["richestjob"] = player.job
+						score["richestkey"] = player.key
+					dmgscore = player.bruteloss + player.fireloss + player.toxloss + player.oxyloss
+					if(dmgscore > score["dmgestdamage"])
+						score["dmgestdamage"] = dmgscore
+						score["dmgestname"] = player.real_name
+						score["dmgestjob"] = player.job
+						score["dmgestkey"] = player.key
 
 	var/nukedpenalty = 1000
-	if (ticker.mode.config_tag == "nuclear")
+	if(ticker.mode.config_tag == "nuclear")
 		var/foecount = 0
 		for(var/datum/mind/M in ticker.mode:syndicates)
 			foecount++
-			if (!M || !M.current)
+			if(!M || !M.current)
 				score["opkilled"]++
 				continue
 			var/turf/T = M.current.loc
-			if (T && istype(T.loc, /area/security/brig)) score["arrested"] += 1
-			else if (M.current.stat == 2) score["opkilled"]++
-		if(foecount == score["arrested"]) score["allarrested"] = 1
+			if(T && istype(T.loc, /area/security/brig))
```

Comment:
Addition to this: the way it works is that any people on the RED flooring are counted as arrested.